src/client/post/PostContent.js

```diff
@@ -181,11 +181,16 @@ class PostContent extends React.Component {
     const postMetaData = jsonParse(content.json_metadata);
     const busyHost = appUrl || 'https://busy.org';
     let canonicalHost = busyHost;
-    if (postMetaData.app && postMetaData.app.indexOf('steemit') === 0) {
+
+    if (_.isString(postMetaData.app)) {
```

Comment:
you don't need this check if you're using` _.indexOf`

Reply:
It looks like this condition should be negated.

It shouldn't matter though if indexOf from lodash can handle non-strings